source/usr/lib/mcvirt/rpc/rpc_daemon.py

```diff
@@ -316,6 +317,10 @@ def register_factories(self):
         libvirt_connector = LibvirtConnector()
         self.register(libvirt_connector, objectId='libvirt_connector', force=True)
 
+        # Create MethodLock object and register with daemon
```

Comment:
This is no longer required, since the deadlock method is part of the ``node`` class